Content team Q4 goals (#13045)
PostHog/posthog.com: contents/teams/content/objectives.mdx

## Editorial

<details>
  <summary>2x-ing product comparisons</summary>

**Owner:** Natalia Amorim

**Motivation:** We want to make product comparisons consistent, up to date, and aligned with our positioning as a developer tool, so they become a reliable entry point for SEO and LLM visibility.

**What we'll ship:**

> * Audit and refresh most popular comparisons to ensure accuracy, consistency, developer-first positioning, LLM readability, etc.  

Reviewer: ```suggestion
* Audit and refresh most popular comparisons to ensure accuracy, consistency, developer-first positioning, LLM readability, etc. Experiment with AI-assisted content (Airops) to improve and speed up this workflow.
```

> * Ship missing error tracking comparisons against key competitors

Reviewer 2: Note from our side:
- Would be great if comparison tables in this content was made reusable somehow. We'd love a version of this in certain places in docs. I think overview is as top funnel as real market pages and this would be a great addition to answer common questions

Reviewer 3: I think @corywatilo is also working on something like this, so if you do, best to coordinate with him.

Reviewer 4: Each product now has a [features array](https://github.com/PostHog/posthog.com/blob/6b6eae68e9d5b8665b082d7809edfe645db4215f/src/hooks/productData/error_tracking.tsx#L232) in json which lists out each competitor and has a value for each of the features.

At some point I want to get the comparison blog posts to reference this data so everything is maintained in the same place.

There are some other things worth considering:

1. How do we limit the features or grouping of features if you want to limit to a subset of features?
2. Does it make more sense to have a single file for each competitor (instead of being split into each of our products) so that we can do more comprehensive competitor comparisons that are by _company_ and not by _product_?

**We'll know we're successful when:**

* Every major competitor has a dedicated, up-to-date comparison article.  
* Comparisons consistently reflect our positioning as a dev tool  
* LLM outputs reflect accurate, up-to-date info in comparison queries.  
* We see steady growth in visibility from both traditional search and LLM outputs.

</details>

<details>

>   <summary>Sharpening our AEO strategy</summary>

Reviewer 2: Letsss gooooooooooooooo. Excited to learn about how GEO/AEO actually works lol. And to stop LLMs from lying/misquoting docs.

Reviewer: Oooh okay is this something we know is currently happening? Currently if it would be worth creating an AEO dashboard for Docs in our reporting tool, but I'm not sure which prompts we'd wanna track

>   <summary>Sharpening our AEO strategy</summary>

Reviewer: Added to the what we'll ship section

**Owner:** Natalia Amorim

**Motivation:** We want to ensure PostHog is represented accurately and prominently in LLM outputs by tracking prompts, monitoring citations, and running experiments to see how optimized and new content can influence the answers these models generate.

**What we'll ship**

* Roll out an AEO tracking tool to monitor prompts and citations at both a general and per-product level.  

> * Run a Q4 “blitz” on error tracking as treatment vs control to measure AEO impact before scaling.  

Reviewer 2: Error tracking owes our team a beer for how much special treatment we've given them. (/s

Reviewer: 100%

  * Create and/or optimize content specifically for LLMs   
  * Pilot YouTube as an AEO lever via prescriptive creator briefs

**We'll know we're successful when**

* We have end-to-end visibility on how PostHog shows up in LLMs across products  
* Key competitor and product prompts consistently surface PostHog in accurate, positive ways and echo our developer tool positioning  
* We can measure the impact of AEO efforts and expand proven tactics across product lines.

</details>

<details>
  <summary>Better reporting on content performance</summary>

**Owner:** Natalia Amorim

> **Motivation:** Right now, performance data is scattered and often looked at in isolation. We want to introduce a structured monthly content performance review to give the team clearer visibility across products, spot trends, and use those insights to guide planning, prioritization, and decision-making.

Reviewer 2: YES (also so many goalsssss

**What we'll ship**

> * A recurring monthly content review  

Reviewer 2: I'd love to get to a point where we can do this for docs content 🤞

* Dashboards and supporting materials

**We'll know we're successful when**

* The team has clear visibility into how content is performing at both an overall and per-product level.  
* Everyone can access and understand key metrics without needing to dig.  
* Insights from reviews consistently shape our content planning and prioritization.

</details>

<details>
  <summary>Newsletter</summary>

**Owner:** Ian Vanagas

**Motivation:** We've scaled back our paid acquisition with the newsletter, so we're going to migrate to Beehiiv. We'll grow more through better integration and analytics. This will also help us tie the newsletter closer to the "marketing funnel".

**What we'll ship:**

- A great newsletter every two weeks  
- Audit of subscriber list to remove inactive subs  

> - Migrate to Beehiiv and set up better onboarding flow, analytics, referrals, better integrated in [posthog.com](http://posthog.com) and other nice features

Reviewer 3: ```suggestion
- Migrate to Beehiiv and set up better onboarding flow, analytics, referrals, better integrated in posthog.com and other nice features
```

Reviewer 2: https?

**We'll know we're successful when:**

> - Newsletter is growing faster than when we left Substack  

Reviewer 2: The move is finally happening :O

- Newsletters getting more views each time

</details>

<details>
  <summary>Better high-level messaging</summary>

> **Owner:** [Ian Vanagas](mailto:ian@posthog.com)

Reviewer 3: email me

> **Owner:** [Ian Vanagas](mailto:ian@posthog.com)

Author: ```suggestion
**Owner:** Ian Vanagas

> **Motivation:** We’ve had a lot of changes in terms of positioning of the company lately and that has left a scattering of messaging across our site, content, and how we talk about ourselves. 

Reviewer 2: Interested to follow this. Docs will largely follow/replicate how brand/marketing/editorial positions messaging

**What we'll ship:**

- A marketing handbook page with key messaging, such as  
  - What is PostHog?  
  - What are the benefits of using PostHog?  
  - Why use PostHog over other options?  
- Update pages that talk about these things to be consistent  
- A statement we can use everywhere

**We'll know we're successful when:**

- Our own team and content is using this messaging more  
- We hear that messaging back from users  
- People on social media use this messaging, instead of "product analytics"

</details>

<details>
  <summary>Ship the product engineer handbook this time</summary>

**Owner:** Andy

> **Motivation:** Same reasons as before

Reviewer 2: We have so many new people, we should probably still include a TL;DR?

> ## Docs

Reviewer 3: From past experience, I suspect y'all have 2 too many goals. Do with this information what you will 🤷

<details>
  <summary>Even better SDK reference docs</summary>

**Owner:** Vincent Ge

**Motivation:** We’ve got all the infrastructure to keep shipping SDK references. Now we want them to be versioned, updated automatically on push to master, and hosted in each SDK’s repo.

**What we'll ship**

* References for Flutter, Android, iOS, GoLang. (still by popularity)  

> * Jobs \+ website code to automatically keep references up to date.  

Reviewer 2: ```suggestion
* CI workflows + website code to automatically keep references up to date.  
```

> * Selectors to view references by version. This is important in the future for big organizations that are slow to upgrade.

Reviewer 2: ```suggestion
* Selectors to view references by version. This is important for teams that choose to remain on older SDK versions.
```

**We'll know we're successful when**

* THE definitive source of truth for human and LLM readers for popular SDKs.  
* Versioned references published when new SDK versions are released, without any human in-the-loop.

</details>

<details>
  <summary>Usable API reference docs</summary>

**Owner:** Vincent Ge

> **Motivation:** LLM based tools need these. Big companies looking to lock in, will want to use these. They don’t need to be pretty, but they can’t be wrong.

Reviewer 2: ```suggestion
**Motivation:** LLM based tools need these. Companies to deeply integrate with PostHog as a developer tool will need these. Doesn't need to be pretty, but information must be accurate.
```

**What we'll ship:** 

* Investigate how we generate SDK specs right now  
* Examine past feedback about APIs  
* Ship code and copy updates to correct *only* the parts that are wrong  
* Make the website API references page slightly prettier

**We'll know we're successful when**

* A developer or LLM can use the API docs without being mislead.

</details>

<details>
  <summary>Next-generation installation instructions</summary>

**Owner:** Vincent Ge

**Motivation:** We now own onboarding, wizard, and docs. They have a few problems:

* They’re inconsistent. The information should be generally the exact same. You should be able to start in onboarding, move to docs, and then to wizard and see the same general instructions/approach. This builds trust and makes installation easier to follow.  

>   * They’re hard to maintain. We have 3 copies of the same information.  

Reviewer 2: ```suggestion
* They’re hard to maintain. We have 3 copies of the same information.  
```

>   * They lack clear checkpoints. No self-checking mechanisms that help readers/users/LLMs understand if they’re succeeding.

Reviewer 2: ```suggestion
* They lack clear checkpoints. No self-checking mechanisms that help readers/LLMs understand if they’re succeeding.
```

**What we'll ship**

* A standardized set of “instructions” for humans and machines. Similar to GitHub Spec-Kit, but less general. [See RFC](https://github.com/PostHog/meta/pull/362)  

>   * A way to define clear checkpoints for humans and LLMs to follow. Including APIs and MCP tools if necessary.  

Reviewer 2: ```suggestion
* A way to define clear checkpoints for humans and LLMs to follow. Including APIs and MCP tools if necessary.  
```

>   * Website, monorepo, and wizard updates where applicable to consume this.

Reviewer 2: ```suggestion
* Website, monorepo, and wizard updates where applicable to consume this.
```

</details>

<details>
  <summary>Next-generation wizard</summary>

**Owner: Danilo Campos

**Motivation**: The wizard’s architecture is a dead-end. It’s too hard to maintain, and it can’t support more ambitious integrations in its current form. The code, prompts and context to support integrations are tightly coupled, making maintenance expensive. Error correction is non-existent, so our success comes down to luck. While that luck is overall good, with 80% or more wizard runs happy and healthy, there’s still thousands of people who end up with a bad outcome.

**What we'll ship**

* A wrapped Claude Code SDK CLI tool that uses our MCP for prompting and context provisioning, able to error correct its output in realtime.  
* Fully-runnable example project code per framework

**We'll know we're successful when**:

* Users can’t believe how deep yet CORRECT our auto-integration is  
* Feedback about poor integration experiences decreases dramatically  
* We can easily support additional product integrations by adding high-level prompts and docs resources

</details>

<details>
  <summary>Make the error tracking docs even better</summary>

> **Owner: [Edwin Lim](mailto:edwin@posthog.com)** 

Author: ```suggestion
**Owner:** Edwin Lim
```

* An updated docs handbook with templates, tools, and style guide criteria for writing PostHog documentation  
* A set of LLM prompts to help quickly generate docs drafts

**We'll know we're successful when**:

* Other product teams want their docs to look like error tracking docs  
* Users mention how good the error tracking docs are

</details>

<details>
  <summary>Update all existing product docs to new format and structure</summary>

**Owner: Edwin Lim, [Vincent Ge], [Danilo Campos]

**Motivation**: Ugh… it’s a hassle, but the product docs should be somewhat consistent. Otherwise we run the risk of product docs drifting apart, especially after another quarter of engineers pushing and shipping. 

**What we’ll ship:** 

* Let’s keep it light and focus on rearranging content with minor rewriting  
* New sidebar structure: Overview, Getting started, Concepts, Guides, AI, and Resources  
* Start here page for each product  
* Overview page for each product

**We'll know we're successful when**:

* People agree the docs look more consistent and organized   
* Product teams are writing within the bounds of the new docs structure

</details>

## Video

<details>

>   <summary>Shoot a "season" of YouTube videos</summary>

Reviewer 3: Between this and "what we'll ship", this feels sort of like two separate goals.

1. Publish the videos already on the roadmap
	- State of the Hog 
	- Hackathon documentary
	- Social clips and cuts for all assets
2. Test a bunch of different ideas to see which formats work
	- Dictator or Tech Bro (from its title, doesn't seem super repeatable, but benefit of the doubt)
	- More YouTubey content ideas with James and Tim TBD (also not super repeatable as relies on James and/or Tim which has been a blocker for us in the past)

Maybe where I am hung up on is I'm not sure how what we plan to do helps us accomplish "We learn what formats work and which ones don't" as these don't seem super repeatable. 

I think we can say "Shoot a season and grow our brand" as an overall goal and outcome, but as it stands I don't think this fulfills the "test and learn what formats work". For that to be the case, I'd love to see something like: "Shoot two episodes of two types of repeatable content like video essays, product explainer, PostHog tutorial" etc. Maybe I am missing context and Dictator vs Tech Bro is this though.

Author: Yeah, was thinking the same thing this morning tbh. This goal is very nascent from thinking after our meeting.

Basically, I think we need to pull together a selection of different things, such as:

- One-off hero content: hackathon doc
- Repeatable fun / brandy things: State of the Hog, Dictator or Tech Bro, one or two others
- Educational content: video essays (probably based on newsletter stuff we've done), tutorials (how to become a max ai power user was my first idea here), etc.

My gut says this will end up being a Q4 + Q1 goal, but motivation here is I don't think releasing stuff like the hackathon doc in isolation will work well. We need to get to a point where we can publish a video a week consistently and generate some momentum.

**Owner:** Alex & Andy

**Motivation:** We want to create PostHog superfans by shipping a bunch of fun, brand building YouTube formats as a season of videos over the course of a month of two, so we can test a bunch of different ideas at once. They'll be shot in a studio over the course of a few days for efficiency.

**What we'll ship:**

* The first ever episode of State of the Hog with James and Tim, which we'll repeat annually.  
* Three episodes of the Hackathon documentary \+ the final cut  
* Dictator or Tech Bro  
* More YouTubey content ideas with James and Tim TBD  

> * Social clips and cuts for all assets

Reviewer 2: Will we experiment with instructional/tutorial type videos at all?

* Consistent engagement on social channels (X, LinkedIn, Reddit, Instagram, YouTube)  
* Positive feedback from product teams and customers

</details>

<details>
  <summary>New demo videos</summary>

**Owner:** Ian and Jordo

**Motivation:** Our existing demo video is completely out of date, but our most popular video. We need a new one.

**What we'll ship:** 

* Introductory platform demo video that explains why you should use PostHog
* Product-specific “modules" we can use in docs, product pages, etc.  
* Intro video \+ modules combined into one video  
* An archived (video, script) that will make updating the demo video in the future easier.

**We'll know we're successful when:** 

* The combined videos we publish get more combined views than the existing demo video.  
* Sales feedback that the demo video is helpful

</details>

## Meta

<details>
  <summary>Hire more awesome people for the content team</summary>

**Owner:** Andy

**Motivation:** We want to do lots of things and we need more people to do them

**What we'll ship:** 

* 2x Docs and tutorial writers   
* 1x newsletter contributor / technical marketer
* A YouTuber  
* Coach talent partner so they can own screening

> 

Reviewer 5: Not in Q4 as our hiring plate is full, but still worth getting the request and spec ready for Q1 at some point anyway! (Could still get a contractor though if that would be helpful, or is it better to just wait?)

**We'll know we're successful when**

* Andy spends less time screening initial candidates  
* We've hired the people we need

</details>

<details>
  <summary>Anti-goals</summary>

* Digital PR (both on the community side with Reddit and GitHub, and on the PR outreach side for publications); it's something that could help us move the needle with LLMs, but not a priority in the short or medium term.  
* Multimedia product comparisons. Could be a strategic advantage, not something we have capacity to take on right now  
* Figuring out long term newsletter strategy  

> * Writing all product docs for all product teams. It’s not practical or efficient, we do better if we help with planning, reviewing, and organizing new docs instead \+ providing assets for making docs writing easier for them.  

Reviewer 2: ```suggestion
* Writing all product docs for all product teams. It’s not practical or efficient, we do better if we help with planning, reviewing, and organizing new docs instead.
```
